docs/content/concepts/system-tables.md

```diff
@@ -182,6 +182,13 @@ SELECT * FROM my_table$ro;
 ### Files Table
 You can query the files of the table with specific snapshot.
 
+{{< hint warning >}}
+
+For tables whose current branch has been modified by the deprecated statement 
```

Comment:
Actually, there is no documentation for 'branch' key. We don't need this sentence.